docs/articles/custom-skins.md

```diff
@@ -0,0 +1,116 @@
+# Custom Skins
+
+Files allows you to customize the application with custom skins by modifying xaml resource dictionaries, this allows users to override the default brushes used throughout the app.
+
+Skins are loaded from the ```Skins``` folder in the app's ```LocalState``` folder. A drop-down (Custom skin) is located in Appearance settings that allows you to select a custom skin.
+
+### Quick start
+
+1. Open your text editor of choice, and paste in the template located below. Then, save the file as ```%userprofile%\AppData\Local\Packages\49306atecsolution.FilesUWP_et10x9a9vyk8t\LocalState\Skins\test1.xaml```.
+
+2. Change some of the color values around.
+
+3. Change the selected skin from the appearance settings page.
+
+Files will now use your custom skin. If you add more skins, you can switch between them using the dropdown.
+
+<details>
```

Comment:
Have you checked that the web version of this page also works correctly?

Reply:
I tested locally, I can't test the template section though until it's deployed.

Reply:
It also works on the website, tested locally.

Out of curiosity, why can't you test until it's deployed?

Reply:
I don't have the repo cloned locally right now 🙃.

Reply:
Haha fair enough.